js/components/evidence/utils.js

```diff
@@ -12,8 +12,18 @@ define(['knockout'], function(ko){
             row.ancestorSplicerCount > 0
         )
     }
+
+    function formatNumberWithCommas(val) {
+      return val.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ",");
```

Comment:
Don’t we have `numeral` library for this?

Reply:
Nice catch - I forgot about that library. I'll update to use that instead.